gradle.properties

```diff
@@ -1,2 +1,3 @@
 version=1.2.0-SNAPSHOT
-kestraVersion=1.1.0-SNAPSHOT
\ No newline at end of file
+kestraVersion=1.1.0-SNAPSHOT
+org.gradle.jvmargs=-Xmx1g -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
@tchiotludo @loicmathieu What do you think about this? 

It unlocks the failing CI situation but maybe there's a memory leak somewhere 😅

Reply:
I would advice to put directly on the actions here: https://github.com/kestra-io/actions/blob/33b106290983a526a31529666259dd2148be28bc/.github/workflows/plugins.yml#L80

``` 
JAVA_OPTS: "-Xmx1g -XX:MaxMetaspaceSize=512m"
```
<img width="887" height="307" alt="image" src="https://github.com/user-attachments/assets/20aeea27-a7b0-4b99-9199-9d0cc13e0cbc" />

I will happen soon on all the plugin, I think the MaxMetaspaceSize is the issue here, and I believe we are close to the limit

Reply:
Done: https://github.com/kestra-io/actions/commit/6e3c058bda3c86086037200cec105960414548ed